src/app/components/our-team/our-team.component.scss

```diff
@@ -145,6 +145,7 @@
           color: $dark-black;
           font-size: $fs-15;
           font-weight: $fw-semibold;
+          padding-left: 30px;
```

Comment:
@n-bernat is there any other way to fix this rather than hard coding padding size? Can you  please give it a try

Reply:
When I was writing an answer to your comment, I wanted to say that there is already a hardcoded padding for the parent:

https://github.com/Cloud-CV/EvalAI-ngx/blob/b03bdc41e34fcab1d4723e8189943137836bf104/src/app/components/our-team/our-team.component.scss#L141-L142

And then I looked closer into it and realized, that this container also has `text-align: right`, so this `padding-left: 50px` is useless.

![prev-comment](https://user-images.githubusercontent.com/23532372/70394747-1f8adf00-19f8-11ea-9fa6-fd1819f1d4d7.png)
(the green part is the padding that I mentioned. As you can see, since the container is aligned to the right, padding doesn't affect anything)

So, if you want I can:
**a)** remove padding proposed in this PR and simply increse padding mentioned above to 65px, so it's actually gonna do something,
or 
**b)** remove both padding proposed in this PR and this mentioned above and set `width: 75%` on a `<div>` that contains name and role in the team.

But,
after applying (a) or (b) there is gonna be a small side-effect: role in the team won't go under the photo when the resolution is between 990 and 1090 (horizontally):

![image](https://user-images.githubusercontent.com/23532372/70395473-4ac4fc80-19ff-11ea-9913-ca8b69e8c528.png)

![image](https://user-images.githubusercontent.com/23532372/70395484-72b46000-19ff-11ea-8b0f-bbbadf777584.png)

To eliminate it, I can for example (assuming that we are gonna go with option b) set `max-width: 155px`, but then I will hardcode something again and you wanted to avoid it.

I don't have any other options to be honest, because everything in user's card has `position: absolute`. I mean, of course I can rebuild entire card if you want, but I am not sure if that won't be overcomplicate this simple task.

Reply:
Actually, I made a mistake - I need to hardcode something, because width of a card isn't constant and size of an image next to it is constant (60x60). Setting width to 70% and max-width to 155px won't do anything. I can simply set width to 155px.

Sorry for that